Source/RMG/UserInterface/Widget/RomBrowserWidget.cpp

```diff
@@ -176,24 +182,6 @@ RomBrowserWidget::RomBrowserWidget(QWidget *parent) : QStackedWidget(parent)
     connect(this->action_SetCoverImage, &QAction::triggered, this, &RomBrowserWidget::on_Action_SetCoverImage);
     connect(this->action_RemoveCoverImage, &QAction::triggered, this, &RomBrowserWidget::on_Action_RemoveCoverImage);
 
-    // configure context menu
```

Comment:
Please revert the changes here, clearing and re-adding the actions in the context menu request slot is unneeded and kinda wasteful

Reply:
Done, will include in next commit. (In order for the "Show/Hide Columns" submenu to keep track of column order and update itself accordingly, the columns submenu has to be rebuilt every time it's pulled up. I didn't know how to clear individual actions, so _that's_ why I set the entire context menu to clear and rebuild itself every time. I've since figured out how to reset the submenu by itself, and I've updated the code accordingly.)